### PR TITLE
V3 doc

### PR DESCRIPTION
**Pull request by neoddish**
### PR includes

- [x] documents, demos

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# CHANGES in AVA 3
+
+## 3.0.0
```
Comment (pddpd) [inline CHANGELOG.md:3]: Do we need a tag@alpha?

File: README.md [hunk 2]
```diff
@@ -86,9 +86,9 @@ It can automatically generates the corresponding Dashboard configurations based
 
 ## Sources
 
-* [Guide](https://ava.antv.vision/en/docs/guide/intro)
-* [API](https://ava.antv.vision/en/docs/api)
-* [Examples](https://ava.antv.vision/en/examples/gallery)
+* [Guide](https://ava.antv.antgroup.com/en/docs/guide/intro)
+* [API](https://ava.antv.antgroup.com/en/docs/api)
+* [Examples](https://ava.antv.antgroup.com/en/examples/gallery)
```
Comment (pddpd) [inline README.md:91]: antgroup.com is primary, antv.vision is backup, does the team agree about the issue?

File: packages/ava-react/package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "name": "AntV",
     "url": "https://antv.vision/"
```
Comment (pddpd) [inline packages/ava-react/package.json:7]: here is one more `antv.vision`

File: packages/ava/package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
     "name": "AntV",
     "url": "https://antv.vision/"
```
Comment (pddpd) [inline packages/ava/package.json:7]: here is one more

File: site/docs/guide/smart-board/intro.en.md [hunk 5]
```diff
@@ -129,7 +129,7 @@ ReactDOM.render(
 
 ## 📖 Documentation
 
-For more usages, please check the [API Reference](https://ava.antv.vision/en/docs/api/smart-board/SmartBoard)
+For more usages, please check the [API Reference](https://ava.antv.antgroup.com/en/docs/api/smart-board/SmartBoard)
```
Comment (pddpd) [inline site/docs/guide/smart-board/intro.en.md:132]: SB folder should be removed cc @LAI-X

File: CHANGELOG.md (hunk 1 above)
Comment (neoddish) [reply CHANGELOG.md:3]: it's a todo, we can modify it before publish

File: site/docs/guide/smart-board/intro.en.md (hunk 5 above)
Comment (neoddish) [reply site/docs/guide/smart-board/intro.en.md:132]: @LAI-X open a new PR for this, plz

File: README.md (hunk 2 above)
Comment (neoddish) [reply README.md:91]: that is fine

File: packages/ava/package.json [hunk 6]
```diff
@@ -4,9 +4,9 @@
   "description": "A framework for automated visual analytics.",
   "author": {
     "name": "AntV",
-    "url": "https://antv.vision/"
+    "url": "http://antv.antgroup.com/"
```
Comment (pddpd) [inline packages/ava/package.json:7]: use https plz

image

**Review by pddpd — APPROVED** [review]
LGTM

**Pull request merged**